backend/src/main/java/de/neuefische/backend/repository/SurfSpotRepo.java

```diff
@@ -20,4 +27,8 @@ public void addSurfSpot(SurfSpot surfSpot){
         spotList.add(surfSpot);
 
     }
+
+    public List<SurfSpot> getSpotList(){
```

Comment:
you already have `getAllSurfSpots` in line22, so I don't think you need to have this method

Reply:
Yes, true :) in my current branch I took it out :)